js/src/features/human-resouce/components/EmployeeLocation/EmployeeLocationList.tsx

```diff
@@ -2,29 +2,29 @@ import React from "react";
 import { Timeline } from "antd";
 import { EmployeeLocation } from "LocationModels";
 import { EmployeeLocationListInner } from "./EmployeeLocationListInner";
+import { isCurrentInterval } from "./helper";
 
 interface Props {
   employeeLocations: EmployeeLocation[];
   openEmployeeLocationEditForm: (val: EmployeeLocation) => void;
-  remove: (id: string) => void;
 }
 
 export const EmployeeLocationList: React.FC<Props> = ({
   employeeLocations,
   openEmployeeLocationEditForm,
-  remove,
 }) => (
   <Timeline mode="left">
     {employeeLocations.map((item) => (
       <Timeline.Item
         key={item.id}
-        label={`${item.startedAt} - ${item.endedAt || 'till now'}`}
-        color={item.endedAt ? "blue" : "green"}
+        label={`${item.startedAt} - ${item.endedAt || "till now"}`}
+        color={
+          isCurrentInterval(item.startedAt, item.endedAt) ? "green" : "blue"
```

Comment:
Why use isCurrentInterval if the simple condition is item.endedAt? "blue": "green"

Reply:
because last location isn't current